### ISSUE TITLE
Builder pane shortcuts don't match

### ISSUE THREAD

**Issue by benoit-pierre** [morinted/plover_dictionary_builder#8]
For example I expect to be able to use Alt+d to activate the "Add and Next" push button, but because a custom shortcut is set (something I only discovered by looking at the code), I can't.

**Comment by panathea** [comment]
If you hover over the button, the alt text shows the shortcut. The readme
also explains the shortcuts.

Is it possible to support a custom and hinted shortcut at the same time?

**Comment by benoit-pierre** [comment]
I don't think you can set more than one shortcut, at least not directly. I think you could setup the extra shortcuts mentioned in the readme via dedicated actions.
